src/well_data.py

```diff
@@ -219,6 +220,7 @@ def signal_to_xlsx_for_sdk(signal_values: np.ndarray, time_stamps: np.ndarray, s
         sheet['E5'] = frames_per_second
```

Comment:
Not a big deal, but we should change this to the sampling period instead of sampling freq eventually

Reply:
would that be 1/fps?

Reply:
Yeah, but we don't need to worry about changing it right now